Lokad.AzureEventStore.Test/Lokad.AzureEventStore.Test.csproj

```diff
@@ -24,17 +24,14 @@
   </ItemGroup>
   
   <!-- When developing locally, comment out this block and uncomment the one below it. -->
-  <ItemGroup>
+  <!--<ItemGroup>
```

Comment:
Comments need to be restored.

Reply:
Done!, You may want to bump up the `<PackageReference>` versions after publishing this new version to NuGet.